Client/src/main/java/gui/panels/boards/PlayingArea.java

```diff
@@ -0,0 +1,49 @@
+package gui.panels.boards;
+
+import gameLogic.CellState;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.HashMap;
+import java.util.Map;
+
+class PlayingArea extends JPanel {
+    final static int weight = 320;
+    final static int height = 320;
+    final static int size = 8;
+
+    Map<Point, CellState> boardState = new HashMap<>();
+
+    PlayingArea(){
+        setEmptyBoard();
+    }
+
+     void setEmptyBoard() {
+        setSize(new Dimension(weight, height));
+        for (int i = 0; i < size; i++) {
+            for (int j = 0; j < size; j++) {
+                boardState.put(new Point(i, j), CellState.EMPTY);
+            }
+        }
+        repaint();
+    }
+
+    @Override
+    public void paint(Graphics g) {
+        Image img = createImage(weight, height);
+
+        Graphics2D g2 = (Graphics2D) img.getGraphics();
+
+        g2.setColor(Color.cyan);
+        g2.fillRect(0, 0, weight, height);
+
+
+        for (int i = 0; i < size; i++) {
```

Comment:
I get that you had to use a bit of impera here, but I'd actually try to name the inner part of the loops as a function taking `i` and `j` as params.